### PR TITLE
Per-group softmax output

### PR DESCRIPTION
**Pull request by goodsong81**
Signed-off-by: Songki Choi <songki.choi@intel.com>

### CODE REVIEW

**Review by JihwanEom — APPROVED** [review]
LGTM

File: mpa/modules/models/heads/custom_hierarchical_linear_cls_head.py [hunk 1]
```diff
@@ -111,9 +111,9 @@ def simple_test(self, img):
         for i in range(self.hierarchical_info['num_multiclass_heads']):
             multiclass_logit = cls_score[:, self.hierarchical_info['head_idx_to_logits_range'][i][0]:
                                             self.hierarchical_info['head_idx_to_logits_range'][i][1]]
+            multiclass_logit = torch.softmax(multiclass_logit, dim=1)
```
Comment (sovrasov) [inline mpa/modules/models/heads/custom_hierarchical_linear_cls_head.py:118]: This is exactly the same way how predictions are done in d-o-r, so I think the changes are correct.

File: mpa/modules/models/heads/custom_hierarchical_linear_cls_head.py [hunk 2]
```diff
@@ -111,9 +111,9 @@ def simple_test(self, img):
...
             multiclass_logits.append(multiclass_logit)
-        multiclass_logits = torch.cat(multiclass_logits, dim=1)
-        multiclass_pred = torch.softmax(multiclass_logits, dim=1) if multiclass_logits is not None else None
+        multiclass_pred = torch.cat(multiclass_logits, dim=1)
```
Comment (sovrasov) [inline mpa/modules/models/heads/custom_hierarchical_linear_cls_head.py]: Else branch is also requires here for safety

**Comment by goodsong81** [comment]
Verified via OTE CLI test.
image
